nautobot/dcim/models/racks.py

```diff
@@ -314,9 +314,7 @@ class Rack(PrimaryModel, StatusModel):
         "outer_depth",
         "outer_unit",
     ]
-    dynamic_group_filter_fields = {
-        "group": "group_id",  # Duplicate filter fields that will be collapsed in 2.0
-    }
```

Comment:
If Im reading this and the DynamicGroup code correctly, after eliminating `group_id` from `RackFilterSet` and `RackFilterForm`, we no longer need this line?